.github/workflows/linux-32bit-build-and-test.yaml

```diff
@@ -142,33 +154,49 @@ jobs:
         set -o pipefail
         export LANG=C.UTF-8
         # PostgreSQL cannot be run as root. So, switch to postgres user.
-        sudo -u postgres make -k -C build installcheck IGNORES="${IGNORES}" \
-          SKIPS="${SKIPS}" PSQL="${HOME}/${PG_INSTALL_DIR}/bin/psql" | tee installcheck.log
+        runuser -u postgres -- \
```

Comment:
This is the most important part of the fix, the rest is mostly cosmetic to sync the 64-bit and 32-bit workflows.